crates/topos-tce-synchronizer/src/lib.rs

```diff
@@ -143,6 +144,12 @@ impl GrpcSynchronizerService for SynchronizerService {
         request: Request<CheckpointRequest>,
     ) -> Result<Response<CheckpointResponse>, Status> {
         let request = request.into_inner();
```

Comment:
I have a dumb question: what actually *is* a checkpoint? an you point me to the docs where it is described? The type is a vector of delivery proofs, which is what confuses me a bit I think.

Reply:
The related RFC is [here](https://github.com/toposware/rfcs/blob/main/rfcs/0003-local-storage.md)